src/library/navigation/stack.js

```diff
@@ -0,0 +1,26 @@
+import { createStackNavigator } from '@react-navigation/stack';
+import React from 'react'
+import Home from '../../containers/Home/layout'
+import { Button } from 'react-native'
+const Stack = createStackNavigator();
+export default function StackScreen() {
+    return (
+        <Stack.Navigator>
+            <Stack.Screen name='Home' component={Home} 
+       
+            options={{
+                
+                headerStyle: { backgroundColor: '#B9D7F2' },
+                title: 'Home', 
+                headerRight: () => (
+                    <Button
+                        onPress={() => console.log('Navigation')}
```

Comment:
In order not to leave the console.log, you can use the Alert.alert('') component, or meanwhile not use the onPress.